DataFormats/L1TGlobal/interface/GlobalAlgBlk.h

```diff
@@ -109,9 +101,12 @@ class GlobalAlgBlk
     //Prescale Column
     int m_preScColumn;
 
+    //Menu UUID
+    int m_menuUUID;
+    int m_firmwareUUID;
```

Comment:
This is a feature of the new library, so I believe we will be able to compare the library as parsed by UTM with this unpacked value.  But Takashi et al can probably answer more precisely.